etc/resources/anvil.json

```diff
@@ -0,0 +1,591 @@
+{
+  "types": [],
+  "domains": [
+    {
+      "domain_name": "Anvil",
+      "domain_code": 5,
+      "identifier_encoding": "anvil",
+      "components": [
+        {
+          "component_name": "Halt",
```

Comment:
Something we need to figure out about hierarchy: I guess these errors would be the same for the main node.
How do we avoid duplication? Should the domain be `EraVM` maybe? Although then it becomes a bit less clear how to share the error definitions between anvil-zksync and core. Hmm..

Reply:
Yes, very much so agree. Unfortunately cannot use EraVM here as its already defined in the error definitions. 

It would be nice to have conformity, a written standard / policy perhaps, that keeps error code component naming conventional?

Reply:
I was thinking more about changing the error schema as it evolves. Writing adapters makes it even more complex.
OTOH once these errors are battle-tested, we can move them to EraVM scope and deprecate them here.